google-http-client/src/main/java/com/google/api/client/http/HttpResponseException.java

```diff
@@ -124,6 +130,24 @@ public HttpHeaders getHeaders() {
    * @since 1.14
    */
   public final String getContent() {
+    if (content == null) {
+      return null;
+    }
+    try {
+      return new String(content, charset.name());
+    } catch (UnsupportedEncodingException exception) {
+      // it would be bad to throw an exception while throwing an exception
+      exception.printStackTrace();
```

Comment:
Is there some RuntimeException we can throw?

Reply:
@sduskis no, we are already in exception case

Reply:
Can we log instead?

Reply:
@sduskis, there is no examples and this library is not logging.

Reply:
For a library we really shouldn't be calling printStackTrace. Maybe we can verify the charset sooner?

Reply:
Please just return null.  We can't print a stack trace.

Reply:
Done. Removed printing stacktrace. Returns null at the end of the method